packages/dev/docs/pages/react-aria/accessibility.mdx

```diff
@@ -104,3 +104,22 @@ All React Aria components are tested across a wide variety of devices, browsers,
 * [NVDA](https://www.nvaccess.org) on Windows in Firefox and Chrome
 * [VoiceOver on iOS](https://www.apple.com/accessibility/iphone/vision/)
 * [TalkBack](https://www.android.com/accessibility/) on Android in Chrome
+
+### False positives
+
+There are a number of known accessibility false positives in React Aria and React Spectrum, currently documented here in our [wiki](https://github.com/adobe/react-spectrum/wiki/Known-accessibility-false-positives).
+These are commonly caught by automated accessibility testing tools and can cause unnecessary noise in your own accessibility audits.
+To facilitate the suppression of these false positives, the data attribute `data-a11y-ignore` is included on the problematic elements
+with the relevant `AXE` rule set as its value. Below is a list of the currently available data selectors and their equivalent `AXE` rules:
+
+```tsx
+rules: [
```

Comment:
might need to wrap this in another `{}` object. the syntax coloring might be confused.